anti-behavioral-analysis/dynamic-analysis-evasion.md

```diff
@@ -71,9 +71,9 @@ The related **Virtualization/Sandbox Evasion ([T1497](https://attack.mitre.org/t
 
 ## References
 
-<a name="1">[1]</a> http://joe4security.blogspot.com/2013/06/overloading-sandboxes-new-generic.html
+<a name="1">[1]</a> https://web.archive.org/web/20201031094517/https://joesecurity.org/blog/4310408827727907098
```

Comment:
It looks like this link is valid: https://joesecurity.org/blog/4310408827727907098 (web.archive.org not needed).

Reply:
Fixed and pushed